src/reactor.cc

```diff
@@ -158,10 +175,25 @@ std::set<cyclus::RequestPortfolio<Material>::Ptr> Reactor::GetMatlRequests() {
   std::set<RequestPortfolio<Material>::Ptr> ports;
   Material::Ptr m;
 
-  int n_assem_order =
-      n_assem_core - core.count() + n_assem_fresh - fresh.count();
+  // second min expression reduces assembles to amount needed until
+  // retirement if it is near.
+  int n_assem_order = n_assem_core - core.count() + n_assem_fresh - fresh.count();
```

Comment:
hi @rwcarlsen, would you mind being stylistically consistent with the variable names in this function, please?